.github/workflows/frontend.yml

```diff
@@ -7,25 +7,22 @@ jobs:
     steps:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-        token: ${{ secrets.CI_TOKEN }}
-    - name: Build Laboratory-Operator image
-      run: cd operators/labInstance-operator/ && docker build -f Dockerfile . 
-      if: github.ref != 'refs/heads/master'
-    - name: Build and Push Laboratory-Operator image
+
+    - name: Website - KubeScore Check
+      run: |
+           docker run -v  ${{ github.workspace }}:/Crownlabs zegl/kube-score score --ignore-test=pod-networkpolicy,container-security-context /Crownlabs/webservice/kubernetes/deployment.yaml /Crownlabs/webservice/kubernetes/svc.yaml /Crownlabs/webservice/kubernetes/ingress-callback.yaml  /Crownlabs/webservice/kubernetes/ingress-content.yaml
+
+    - name: Build and Push Frontend image (Dev)
```

Comment:
Shouldn't this image be the pre-prod version instead of dev?

Reply:
AFAIK, the image used in pre-prod are images built from master branch. This image is used only by developers pipelines.

Reply:
Yes, you are right, I didn't notice the if condition ``github.ref != 'refs/heads/master'``. Maybe in the future, we could move this step to the PR pipeline,  for the sake of clarity, if you agree.